src/education.njk

```diff
@@ -22,7 +22,7 @@ hubspot:
                         FlowFuse <span class="text-indigo-600 inline-block">for Education</span>
                     </h1>
                     <p class="mb-12">
-                        FlowFuse is your partner in teaching engineering, IoT, development, OT/IT integration, and automation to your students. Free for educators and students. Contact us for an educational license.
+                        FlowFuse is your partner in teaching engineering, IoT, development, OT/IT integration, and automation to your students. FlowFuse offers a comprehensive suite of tools and resources for accredited educational institutions to accelerate learning, teaching, and research in Node-RED and industrial automation. Instructors and educational administrators are invited to contact us for an educational license, which will supply you and your students with 5 free months of FlowFuse Cloud Starter. 
```

Comment:
Hi @gstout52, do you think we could split it into two paragraphs? It just feels like a big block of text.

Reply:
@Yndira-E > Hi @gstout52, do you think we could split it into two paragraphs? It just feels like a big block of text.

Good idea - should I add something like this to split it up from "Instructors" onward?

  <p>
                        Instructors and educational administrators are invited to contact us for an educational license, which will supply you and your students with 5 free months of FlowFuse Cloud Starter. 
                    </p>

Reply:
I've just took care of it 😉